sci-libs/mkl-rt/mkl-rt-2019.4.243.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils
+
+DESCRIPTION="Intel Math Kernel Library (Runtime)"
+HOMEPAGE="https://software.intel.com/en-us/mkl"
+SRC_URI="https://repo.continuum.io/pkgs/main/linux-64/mkl-2019.4-243.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="ISSL"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="eselect-ldso"
+
+# MKL uses Intel/LLVM OpenMP by default.
+# One can change the threadding layer to "gnu" or "tbb" through the MKL_THREADING_LAYER env var.
+RDEPEND="eselect-ldso? ( !app-eselect/eselect-cblas
+                         =app-eselect/eselect-blas-0.2 )
```

Comment:
This should be `>=app-eselect/eselect-blas-0.2`,  we may revise it for the future.